### PR TITLE
Feature/us64 create post for a need

### PR DESCRIPTION
**Pull request by jpc11623**

### CODE REVIEW

File: Gateway/CorkboardAPI/CorkboardAPI/Controllers/PostsController.cs [hunk 1]
```diff
@@ -8,16 +8,20 @@
...
+using System.Reflection;
 using MongoDB.Bson;
-using System.Threading.Tasks;
 using System.Web.Http.Cors;
+using System.Web.Http.Description;
+using CorkboardAPI.Security;
+using log4net;
+using Newtonsoft.Json;
 
 namespace CorkboardAPI.Controllers
```
Comment (IngageMaddoxT) [inline Gateway/CorkboardAPI/CorkboardAPI/Controllers/PostsController.cs]: This controller feels a little heavy on code.  Consider refactoring into a service.

File: website/app/corkboard/directives/needPostDetails.directive.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+(function () {
+    "use strict()";
+    module.exports = NeedPostDetails;
+
+    NeedPostDetails.$inject = ['$log', '$rootScope', '$timeout', '$window', 'CorkboardListings', 'CORKBOARD_EVENTS'];
+
+    function NeedPostDetails($log, $rootScope, $timeout, $window, CorkboardListings, CORKBOARD_EVENTS) {
+        return {
+            restrict: "EA",
+            //transclude: true,
```
Comment (IngageMaddoxT) [inline website/app/corkboard/directives/needPostDetails.directive.js]: remove commented code

File: Gateway/CorkboardAPI/CorkboardAPI/Controllers/PostsController.cs (hunk 1 above)
Comment (jpc11623) [inline Gateway/CorkboardAPI/CorkboardAPI/Controllers/PostsController.cs]: Discussed, going to be refactored for the next story.

File: website/app/corkboard/directives/needPostDetails.directive.js (hunk 2 above)
Comment (jpc11623) [inline website/app/corkboard/directives/needPostDetails.directive.js]: Removed

**Pull request merged**